common/flatpak-dir.c

```diff
@@ -14068,27 +14089,24 @@ flatpak_dir_get_default_locale_languages (FlatpakDir *self)
   if (accounts_proxy != NULL)
     get_locale_langs_from_accounts_dbus (accounts_proxy, langs);
 
-  /* If none were found, fall back to using all languages */
-  if (langs->len == 0)
-    return g_new0 (char *, 1);
```

Comment:
Why are you deleting this? If we have no account service, and if extra-languages is not set, i don't think using only the current locale is the right thing. That may be different to the thing someone eventually uses. For example locale might be C or en_US during a system install, but then change to something else.

Reply:
If `extra-languages` is unset and no languages were found through localed or AccountsService, we are still using all languages. In that case the `flatpak_strv_merge` below will return an empty array.

Reply:
You're right, will remove this change.

Reply:
No I was explaining why I think we should should leave this `if (langs->len == 0) return g_new0 (char *, 1);` deleted.